pkg/products/bamboo/locals.tf

```diff
@@ -27,4 +27,25 @@ locals {
 
   ingress_settings   = local.use_domain ? local.ingress_with_domain : local.service_as_loadbalancer
   storage_class_name = "efs-cs"
+
+  license_settings = yamlencode({
+    bamboo = {
+      license = {
+        secretName = kubernetes_secret.license_secret.metadata[0].name
+        secretKey  = "license"
+      }
+    }
+  })
+
+  admin_settings = yamlencode({
+    bamboo = {
+      sysadminCredentials = {
```

Comment:
We may not need to put all this items in the secret.

Reply:
The helm chart is expecting it being in the secret - https://github.com/atlassian/data-center-helm-charts/blob/main/src/main/charts/bamboo/values.yaml#L398

Reply:
As from Bamboo Helm chart, it requires all values stored in the secret. 
https://github.com/atlassian/data-center-helm-charts/blob/main/src/main/charts/bamboo/values.yaml#L398